src/scripts/libraryBrowser.js

```diff
@@ -90,6 +90,10 @@ export function getQueryPagingHtml (options) {
         html += '<span style="vertical-align:middle;">';
         html += globalize.translate('ListPaging', totalRecordCount ? startIndex + 1 : 0, recordsEnd, totalRecordCount);
         html += '</span>';
+    } else {
```

Comment:
Similar thing here, but I think we can remove the check entirely and always use the logic above.

Reply:
It should equal the `totalRecordCount` I think.

```js
const recordsEnd = Math.min(startIndex + limit, totalRecordCount)
```

Reply:
No it's, not, as it's the `Math.min(..., ...)` and the `startIndex + limit` equals to `0` when no paging is enabled.